x/revenue/module.go

```diff
@@ -6,6 +6,8 @@ import (
 	"fmt"
 	"math/rand"
 
+	"cosmossdk.io/errors"
```

Comment:
Nit: refactor to be consistent with codebase
```suggestion
	errorsmod "cosmossdk.io/errors"
```